CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 0.4.0
+
+* [FEATURE] Add counter for errors and change the entry point of exporter metrics (#7)
+* [FEATURE] New easy-to-write register addresses (#23)
+* [FEATURE] Implement exporter-toolkit (#42)
+* [FEATURE] Overhaul build, lint, Docker, and CI (#47)
+* [ENHANCEMENT] Adjusted the typo fix (#24)
+* [ENHANCEMENT] Corrected various typos in docs, comments, and diagrams
+* [ENHANCEMENT] Cleanup uncessary metrics (#21)
+* [ENHANCEMENT] Improve and clarify example (#26)
+* [ENHANCEMENT] Set metrics_path in prometheus.yml (#31)
+* [ENHANCEMENT] Remove stale license information(#33)
+* [ENHANCEMENT] README: mention that sub_module is a required parameter and target needs to include the port (#36)
+* [ENHANCEMENT] Cleanup build (#43)
+* [ENHANCEMENT] Fix README Diagram (#45)
+* [ENHANCEMENT] Fix another diagram typo (#48)
+* [ENHANCEMENT] Fix CodeQL Error (#50)
```

Comment:
We typically only list user visible changes. We don't need to include stuff like test fixes or build stuff.

Reply:
Fixed!